dicoogle/src/main/java/pt/ua/dicoogle/server/users/UserExportPresets.java

```diff
@@ -66,18 +65,24 @@ public static void savePreset(String username, String presetName, String[] field
      * @throws IOException if an I/O error occurs
      */
     public static Map<String, String[]> getPresets(String username) throws IOException {
-        File presetsDir = new File(Platform.homePath() + "users/" + username + "/presets/");
+        Path presetsDir = Paths.get(Platform.homePath(), "userdata", username, "presets");
 
         Map<String, String[]> presets = new HashMap<>();
 
-        if (!presetsDir.isDirectory()) {
+        if (!Files.isDirectory(presetsDir)) {
             return presets;
         }
 
-        for (File presetFile : presetsDir.listFiles()) {
-            String[] fields = FileUtils.readFileToString(presetFile, "UTF-8").split("\n");
-            presets.put(FilenameUtils.removeExtension(presetFile.getName()), fields);
-        }
+        Stream<Path> fileList = Files.list(presetsDir);
+        fileList.filter(file -> !Files.isDirectory(file))
+                .forEach(file -> {
+                    try (ObjectInputStream in = new ObjectInputStream(Files.newInputStream(file))) {
```

Comment:
Please consider rolling back to a plain text file. Obtaining a stream of lines from a file is as easy as [Files#lines](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#lines-java.nio.file.Path-java.nio.charset.Charset-).